exercise2/src/Controller/CalculatorController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace src\Controller;
+
+use src\Model\CalculatorModel;
+
+class CalculatorController 
+{
+
+    public function getResult()
+    {
+        if(isset($_POST['num1'])) 
+        {
```

Comment:
NIT. It's better if you inline opening curly braces for this kind of statements.